content/management/configuration-api/v3.2/index.mdx

```diff
@@ -2785,6 +2785,236 @@ curl -X POST \
 
 </Section>
 
+# Auto access
+
+**Auto access** allows you to assign groups to a thread that will be created when a customer starts a chat. Those groups are
+assigned based on the conditions defined to match the customer's tracking URL and geolocation. Every time a customer calls
+**Login** or **Update Customer Page**, our backend checks if the tracking URL and geolocation match the pre-defined conditions.
+Given there's a match, the groups are assigned to a thread, but not until a chat starts and the thread is actually created.
+
+If a customer starts a chat via **Start Chat** or **Activate Chat** with the `chat.access` parameter, `chat.access` overrides the auto access
+assigned to the thread.
+
+## Methods
+
+<Section>
+
+<Text>
+
+### Add auto access
+
+Creates an `auto access` data structure, which is a set of conditions for the tracking URL and geolocation of a customer.
+
+#### Specifics
+
+|                |                                                                               |
+| -------------- | ----------------------------------------------------------------------------- |
+| Method URL     | `https://api.livechatinc.com/v3.2/configuration/action/add_auto_access`       |
+| Required scopes| `access_rules:rw`                                                             |
+
+
+| Parameter       | Required     | Data type | Notes                         |
+| --------------- | ------------ | --------- | ----------------------------- |
+| `access`        | Yes          | `object`  | Destination access            |
+| `access.groups` | Yes          | `[]int`   | Destination groups            |
+| `conditions`__**__   | Yes          | `object`  | Rules to check                |
```

Comment:
Since the naming changed `rule` -> `condition`, I think we should also replace "rules" with "conditions" in the parameter descriptions